include/nuttx/lin.h

```diff
@@ -0,0 +1,91 @@
+/****************************************************************************
+ * include/nuttx/lin.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __INCLUDE_NUTTX_LIN_H
+#define __INCLUDE_NUTTX_LIN_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <nuttx/can.h>
+
+#ifdef CONFIG_NET_CAN
```

Comment:
Looks good, but one question here. Why do we need this ifdef wrapper? Is "classic" LIN (same as CAN) driver going to be supported in parallel?

Reply:
This is indeed a problem, and I have corrected it！